### DISCUSSION TITLE
Filter by visit duration

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#1790] [Anything Else]
"One of the metrics we use internally is the number of visitors who spent in excess of 1 minute browsing the site. We’d like to be able to filter by visit duration > 1 minute"

(request via email)

**Comment by kiru** [comment]
Is it also possible to expose that number in the dashboard? For example, a histogram of the visit duration? Even a mean/avg/std numbers would be useful.

**Comment by jasper72** [comment]
Yes, I'd love to filter out the many bots that presumably represent the 0s visits.
